<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>661</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Object Maker</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget_sprites">
      <property name="enabled">
       <bool>true</bool>
      </property>
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">QTabWidget::pane {
  border: 1px solid lightgray;
  top:-1px; 
  background: rgb(245, 245, 245);; 
} 

QTabBar::tab {
  background: rgb(230, 230, 230); 
  border: 1px solid lightgray; 
  padding: 5px;
} 

QTabBar::tab:selected { 
  background: rgb(245, 245, 245); 
  margin-bottom: -1px; 
}</string>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_sprite">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <attribute name="title">
        <string>Sprite 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2"/>
      </widget>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QWidget" name="widget_spriteExchange" native="true">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>32</width>
        <height>0</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>2</number>
       </property>
       <property name="rightMargin">
        <number>2</number>
       </property>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Minimum</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>24</width>
           <height>125</height>
          </size>
         </property>
        </spacer>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QToolButton" name="toolButton_toSprites">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>5</width>
           <height>16</height>
          </size>
         </property>
         <property name="text">
          <string/>
         </property>
         <property name="arrowType">
          <enum>Qt::LeftArrow</enum>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter">
        <widget class="QToolButton" name="toolButton_lock">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>28</width>
           <height>19</height>
          </size>
         </property>
         <property name="text">
          <string>Lck</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item alignment="Qt::AlignHCenter|Qt::AlignTop">
        <widget class="QToolButton" name="toolButton_fromSprites">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>...</string>
         </property>
         <property name="arrowType">
          <enum>Qt::RightArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="3">
     <widget class="QTabWidget" name="tabWidget_objects">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>0</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">
QTabWidget::pane {
  border: 1px solid lightgray;
  top:-1px; 
  background: rgb(245, 245, 245);; 
} 

QTabBar::tab {
  background: rgb(230, 230, 230); 
  border: 1px solid lightgray; 
  padding: 5px;
} 
QTabBar::tab:selected { 
  background: rgb(245, 245, 245); 
  margin-bottom: -1px; 
}
QToolButton {background : lightgrey;
border: 1px solid darkgrey;}
</string>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <property name="documentMode">
       <bool>false</bool>
      </property>
      <property name="tabsClosable">
       <bool>true</bool>
      </property>
      <property name="movable">
       <bool>false</bool>
      </property>
      <property name="tabBarAutoHide">
       <bool>false</bool>
      </property>
      <widget class="QWidget" name="tab_object">
       <attribute name="title">
        <string>Object 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_3"/>
      </widget>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QGroupBox" name="groupBox_tools">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>280</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>310</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="title">
       <string>Tools</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <property name="topMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QWidget" name="widget_tools_container" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QWidget" name="widget_tools" native="true"/>
          </item>
          <item>
           <widget class="Line" name="line">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QGroupBox" name="groupBox_tool_settings">
            <property name="title">
             <string>Brush Options</string>
            </property>
            <property name="flat">
             <bool>true</bool>
            </property>
            <widget class="QDial" name="dial_Brushsize">
             <property name="geometry">
              <rect>
               <x>0</x>
               <y>20</y>
               <width>61</width>
               <height>61</height>
              </rect>
             </property>
             <property name="minimum">
              <number>1</number>
             </property>
             <property name="maximum">
              <number>10</number>
             </property>
             <property name="tracking">
              <bool>false</bool>
             </property>
             <property name="invertedAppearance">
              <bool>false</bool>
             </property>
             <property name="wrapping">
              <bool>false</bool>
             </property>
             <property name="notchesVisible">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_brushSolid">
             <property name="geometry">
              <rect>
               <x>64</x>
               <y>20</y>
               <width>51</width>
               <height>21</height>
              </rect>
             </property>
             <property name="text">
              <string>Solid</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
            <widget class="QPushButton" name="pushButton_brushAirbrush">
             <property name="geometry">
              <rect>
               <x>64</x>
               <y>50</y>
               <width>50</width>
               <height>20</height>
              </rect>
             </property>
             <property name="text">
              <string>Airbrush</string>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_selectedColor">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>272</width>
           <height>239</height>
          </size>
         </property>
         <property name="title">
          <string>Selected colors</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <property name="spacing">
           <number>3</number>
          </property>
          <property name="leftMargin">
           <number>0</number>
          </property>
          <property name="topMargin">
           <number>0</number>
          </property>
          <property name="rightMargin">
           <number>0</number>
          </property>
          <property name="bottomMargin">
           <number>0</number>
          </property>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_Colormani">
         <property name="title">
          <string>Color Manipulations (selected colors are affected)</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0" colspan="2">
           <widget class="QCheckBox" name="checkBox_allViews">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Apply to all views</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QPushButton" name="pushButton_remapTo">
            <property name="text">
             <string>Remap to</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="comboBox_remapToColor"/>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButton_incrBrightness">
            <property name="text">
             <string>+ Brightness</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_decrBrightness">
            <property name="text">
             <string>- Brightness</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButton_deleteColor">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_Brushes">
         <property name="title">
          <string>Brushes</string>
         </property>
         <property name="flat">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1400</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Object</string>
    </property>
    <widget class="QMenu" name="menuLoad_Object_from">
     <property name="title">
      <string>Open Object from</string>
     </property>
     <addaction name="actionOpenFile"/>
     <addaction name="actionDATIdentifier"/>
    </widget>
    <widget class="QMenu" name="menuNew_Object">
     <property name="title">
      <string>New Object</string>
     </property>
     <addaction name="actionSmallScenery"/>
     <addaction name="actionLargeScenery"/>
     <addaction name="actionWall"/>
    </widget>
    <addaction name="menuNew_Object"/>
    <addaction name="menuLoad_Object_from"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveObjectAt"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Sprite</string>
    </property>
    <addaction name="actionNewSprite"/>
    <addaction name="actionLoadImage"/>
   </widget>
   <widget class="QMenu" name="menuSettings">
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QMenu" name="menuImport_Color">
     <property name="title">
      <string>Transparency Color</string>
     </property>
     <addaction name="actionBlack"/>
     <addaction name="actionWhite"/>
     <addaction name="actionUpper_Left_Pixel"/>
     <addaction name="actionCustom_Color"/>
    </widget>
    <widget class="QMenu" name="menuPalette">
     <property name="title">
      <string>Palette</string>
     </property>
     <addaction name="actionPaletteOpenRCT"/>
     <addaction name="actionPaletteOld"/>
    </widget>
    <addaction name="actionSettings"/>
    <addaction name="menuImport_Color"/>
    <addaction name="menuPalette"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuTools"/>
   <addaction name="menuSettings"/>
  </widget>
  <action name="actionSaveObjectAt">
   <property name="text">
    <string>Save Object at...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save Object</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionOpenFile">
   <property name="text">
    <string>File...</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDATIdentifier">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>DAT Identifier...</string>
   </property>
  </action>
  <action name="actionNewSprite">
   <property name="text">
    <string>New Image</string>
   </property>
  </action>
  <action name="actionLoadImage">
   <property name="text">
    <string>Open Image</string>
   </property>
  </action>
  <action name="actionSmallScenery">
   <property name="text">
    <string>Small Scenery</string>
   </property>
  </action>
  <action name="actionLargeScenery">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Large Scenery</string>
   </property>
  </action>
  <action name="actionWall">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Wall</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings...</string>
   </property>
  </action>
  <action name="actionBlack">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Black (0,0,0)</string>
   </property>
  </action>
  <action name="actionWhite">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>White (255,255,255)</string>
   </property>
  </action>
  <action name="actionUpper_Left_Pixel">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Upper Left Pixel</string>
   </property>
  </action>
  <action name="actionCustom_Color">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Custom Color</string>
   </property>
  </action>
  <action name="actionPaletteOpenRCT">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Open RCT</string>
   </property>
  </action>
  <action name="actionPaletteOld">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Green Remap</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
